scripts/tasks.bash

```diff
@@ -83,6 +83,17 @@ function build-code-server() {
 	cd "${buildPath}/out/vs/server" && yarn --production --ignore-scripts
 	rm "${buildPath}/out/vs/server/"{package.json,yarn.lock}
 
+	# onigasm 2.2.2 has a bug that makes it broken for PHP files so use 2.2.1.
+	# https://github.com/NeekSandhu/onigasm/issues/17
+	local onigasmPath="${buildPath}/node_modules/onigasm-umd"
+	rm -rf "${onigasmPath}"
+	git clone "https://github.com/alexandrudima/onigasm-umd" "${onigasmPath}"
+	cd "${onigasmPath}" && yarn && yarn add --dev onigasm@2.2.1 && yarn package
+	mkdir "${onigasmPath}-temp"
```

Comment:
You should add ``-p`` so that it does not fail when the directory already exists.

Reply:
I don't believe it's possible for this directory to exist because
we remove and create the parent directory, copy the
`package.json`, then run `yarn`. That means the only way for it to
exist would be if there was a module called `onigasm-umd-temp` and
the `package.json` listed it as a dependency. If that were the
case, we wouldn't want to copy to or from it (since it would be a
valid module) and failing on the `mkdir` would be preferable.

Reply:
Anything in that list could fail and leave that directory behind.

Reply:
We remove the parent directory so once the `mkdir` line is reached that directory won't exist unless it was created by `yarn`.

https://github.com/cdr/code-server/blob/6737384d27f70f6dbc5cfcd1f87301ca6c27fa1f/scripts/tasks.bash#L69-L70

Reply:
You can resolve this if this is a invalid suggestion @code-asher. I do agree this is unnecessary since we already define it at those parts.